src/actions/ticket-actions.js

```diff
@@ -853,7 +853,9 @@ export const getTicketTypes =
       page: currentPage,
       per_page: perPage,
       access_token: accessToken,
-      expand: "badge_type"
+      expand: "badge_type",
+      relations: "badge_type,badge_type.none",
+      fields: "badge_type.name,badge_type.id"
```

Comment:
i do think that we could restrict more and remove more fields
* created
* last_edited
* summit_id
* currency
* max_quantity_per_order
* quantity_sold
* sub_type